<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>BPEL2oWFN</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <property name="elideMode" >
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons" >
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Single BPEL process</string>
       </attribute>
       <attribute name="icon" >
        <iconset>icons/bpel.png</iconset>
       </attribute>
       <attribute name="toolTip" >
        <string>Translation of a single BPEL process into a Petri net model.</string>
       </attribute>
       <widget class="QPushButton" name="pushButton" >
        <property name="geometry" >
         <rect>
          <x>220</x>
          <y>200</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text" >
         <string>PushButton</string>
        </property>
        <property name="autoDefault" >
         <bool>false</bool>
        </property>
        <property name="default" >
         <bool>false</bool>
        </property>
        <property name="flat" >
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2" >
        <property name="geometry" >
         <rect>
          <x>400</x>
          <y>160</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text" >
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="reduction_groupBox" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>260</y>
          <width>231</width>
          <height>81</height>
         </rect>
        </property>
        <property name="title" >
         <string>Structural reduction</string>
        </property>
        <widget class="QSlider" name="reduction_horizontalSlider" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>30</y>
           <width>211</width>
           <height>27</height>
          </rect>
         </property>
         <property name="maximum" >
          <number>6</number>
         </property>
         <property name="pageStep" >
          <number>1</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="reduction_label" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>50</y>
           <width>211</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text" >
          <string>none</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="commandLine_groupBox" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>361</y>
          <width>631</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title" >
         <string>Command line</string>
        </property>
        <widget class="QLineEdit" name="commandLine_lineEdit" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>30</y>
           <width>611</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text" >
          <string>bpel2owfn</string>
         </property>
         <property name="frame" >
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="commandLine_checkBox" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>60</y>
           <width>241</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text" >
          <string>use long command-line options</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="" >
        <property name="geometry" >
         <rect>
          <x>190</x>
          <y>100</y>
          <width>100</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QRadioButton" name="radioButton" >
           <property name="text" >
            <string>RadioButton</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2" >
           <property name="text" >
            <string>RadioButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_3" >
           <property name="text" >
            <string>RadioButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>BPEL4Chor choreography</string>
       </attribute>
       <attribute name="icon" >
        <iconset>icons/bpel4chor.png</iconset>
       </attribute>
       <attribute name="toolTip" >
        <string>Translation of a BPEL4Chor choreography (participant descriptions and participant topology) into a Petri net model.</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3" >
       <attribute name="title" >
        <string>Static Analysis</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <tabstops>
  <tabstop>reduction_horizontalSlider</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <includes/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
